lib/LDAPUserManager.php

```diff
@@ -217,6 +217,9 @@ public function createUser($username, $password) {
 			throw new Exception('Acting user is not from LDAP');
 		}
 		try {
+			if (is_null($adminUser)) {
```

Comment:
please add a comment about the use case (that null as adminUser can be valid), and use `$adminUser === null` as comparison.